Applications/WebPortal/WebApp/library/source/pages/account_directory_page/create_account_page.tsx

```diff
@@ -400,19 +456,20 @@ export class CreateAccountPage extends React.Component<Properties, State> {
 
   private onSubmit() {
     if(this.checkInputs()) {
-      this.props.onSubmit(this.state.username, this.state.groups,
+      this.props.onSubmit(this.state.username, this.state.selectedGroups,
         this.state.identity, this.state.roles);
     }
   }
 
   private checkInputs(): boolean {
-    const errorFirstName = this.state.identity.firstName === '';
-    const errorLastName = this.state.identity.lastName === '';
-    const errorUsername = this.state.username === '';
-    const errorEmail = this.state.identity.emailAddress === '' ||
+    const hasErrorFirstName = this.state.identity.firstName === '';
+    const hasErrorLastName = this.state.identity.lastName === '';
+    const hasErrorUsername = this.state.username === '';
+    const hasErrorGroups = this.state.selectedGroups.length < 1;
+    const hasErrorEmail = this.state.identity.emailAddress === '' ||
       (!this.state.identity.emailAddress.includes('@') &&
         !this.state.identity.emailAddress.includes('.'));
```

Comment:
Don't perform any e-mail address validation other than testing for the empty string. It's not possible to validate an e-mail address.